pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -254,6 +254,26 @@ class StateVectorCudaManaged
         }
     }
 
+    /**
+     * @brief Apply a single gate to the state vector.
+     *
+     * @param opName Name of gate to apply.
+     * @param controlled_wires Control wires.
+     * @param wires Wires to apply gate to.
+     * @param inverse Indicates whether to use adjoint of gate.
+     * @param params Optional parameter list for parametric gates.
+     * @param params Optional std gate matrix if opName doesn't exist.
+     */
+    void applyOperation(const std::string &opName,
+                        const std::vector<size_t> &controlled_wires,
+                        const std::vector<size_t> &wires, bool inverse = false,
+                        const std::vector<Precision> &params = {0.0},
+                        const std::vector<CFP_t> &gate_matrix = {}) {
+        PL_ABORT_IF(!controlled_wires.empty(),
+                    "Controlled kernels not implemented.");
+        applyOperation(opName, wires, inverse, params, gate_matrix);
+    }
```

Comment:
Do you know why coverage is not reaching this point?

Reply:
Because `pennylane_lightning/core/src/simulators/base/tests/Test_StateVectorBase.cpp` does not include L-GPU. This will be solved when L-GPU's complex types are brought in line with L-Qubit's and L-Kokkos'.